### ISSUE TITLE
Improve the "lessons" section

### ISSUE THREAD

**Issue by FrancescoXX** [FrancescoXX/rustcrab#164]
improve the "lessons" section

https://www.rustcrab.com/lessons

if possible, find a way to automatically fetch all the videos in the playlist and add them automatically

Also , add a number to the lesson somewhere (small)

**Comment by Muhammad-Owais-Warsi** [comment]
So basically it's like fetching from the youtube api, if am not wrong.

**Comment by mohitjoping** [comment]
Hi @FrancescoXX, hope you're doing well! I'm a front-end developer with experience in creating user-friendly and visually appealing UIs. I've built a few projects using TypeScript, and I feel confident I can handle this issue. I'd love the opportunity to work on it!

**Comment by FrancescoXX** [comment]
hey @mohitjoping , the design looks cool, but here let's focus on fetching the lessons. I suggest you to create another issue for this

**Comment by FrancescoXX** [comment]
> So basically it's like fetching from the youtube api, if am not wrong.

yes, but in case we can't fetch them all automatically they need to be updated manually

**Comment by Suraj-kumar00** [comment]
Hi @FrancescoXX, Can I work on this issue?
